packages/webapp/src/plugins/redirect-view.js

```diff
@@ -0,0 +1,37 @@
+import constants from '../utils/constants.js'
+
+const redirectView = {
+  name: 'redirect-view',
+  register: (server, _options) => {
+    server.decorate('toolkit', 'redirectView', function (route, data) {
+      if (this.request.isInjected && route !== '/test') {
```

Comment:
isInjected is where we inject via a test - the exception here is the /test route which is used by the redirect-view.spec.js test to assert the redirection happens. i.e. we don't want a redirect for the existing error view tests on existing routes.